ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml

```diff
@@ -88,6 +88,22 @@ tests:
     - ref: openstack-test-openstack
     workflow: openshift-e2e-openstack-ipi
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-openstack-dualstack
```

Comment:
I'm being picky but it'll be nice to name is dualstack-v4primary to avoid confusion. You an I know it's the case but anyone else could be mislead.

Reply:
Let's update these jobs and all the other existent dualstack jobs in a separate PR so all have the same name. Does that work for you?

Reply:
yes